spec/demo_cheet_spec.rb

```diff
@@ -0,0 +1,28 @@
+# require 'spec_helper'
+
+describe Cheet do
```

Comment:
all the tests dont run because of this constant Cheet. Should you be requiring that file?